hash/ibm5150.xml

```diff
@@ -2937,6 +2949,29 @@ Known PC Booter Games Not Dumped, Or Dumped and Lost when Demonlord's Site went
 			</dataarea>
 		</part>
 	</software>
+	
+	<software name="drafix">
+		<description>Drafix Windows CAD (Demo)</description>
+		<year>1990?</year>
+		<publisher>Fox Data</publisher>
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="1228800">
+				<rom name="drafix windows cad (demo).img" size="1228800" crc="95f50d9f" sha1="bc9de150a22e58a8ad4a0948d66f203527d84f32" offset="0" />
+			</dataarea>
+		</part>
+	</software>
+	
+	<software name="easypc">
+		<description>Easy-PC Demonstrator for Printed Circuit Boards and Schematics</description>
+		<year>19??</year>
+		<publisher>Number One Systems</publisher>
+		<info name="version" value="Rev. I0123" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="368640">
+				<rom name="easy-pc demonstrator for printed circuit boards and schematics (rev i0123).img" size="368640" crc="ab548445" sha1="1bb4cc9d514eb2caa537a8445e5b84a9409631c7" status="baddump" offset="0" />
```

Comment:
What is bad about the dump? Comments are helpful.

Same for mforsta below.

Reply:
i just try to help/solve that thingy

mastering borland c plus plus 3.1
https://www45.zippyshare.com/v/kBKp1K2W/file.html
drafix windows cad (demo)
https://www45.zippyshare.com/v/Pct7ioCL/file.html
tango printed circuit design evaluation
https://www45.zippyshare.com/v/PqzdWmEt/file.html
tango schematic capture evaluation
https://www45.zippyshare.com/v/r7TF2LpG/file.html
these ones with default settings converted back to img

but the other ones gave with default settings different crc`s

Reply:
the first converted was using default settings and the rest was using 360rpm

Reply:
2630 Fourier Analyzer
https://www54.zippyshare.com/v/Hj0Xw1N4/file.html
Mitt första ordbehandlingsprogram
https://www54.zippyshare.com/v/0cZnqZx4/file.html
Easy PC Demonstrator For Printed Circuit Boards And Schematics
https://www54.zippyshare.com/v/vf3a7v4U/file.html
The Great International Paper Airplane Construction Kit
https://www54.zippyshare.com/v/NDr6IYL1/file.html

last dump Ranger 1 is really weird ,2 can be converted but one file dont want to be converted it show everytime the wrong crc,used rpm was 300 and 360 both are not working for that one file

Reply:
last one converted by kryoflux version 2.51 -360rpm
ranger1
https://www19.zippyshare.com/v/Ryoj3N3H/file.html

Reply:
Thank you @MdGheist <3
Work filled up my time so I couldn't get around to uploading anything until today, but I see you've taken care of that already.
I'm on Linux, so the latest DTC version I have is 2.60. That's what I use to convert.

@DopefishJustin 
I actually wrote these software additions so long ago I can't remember what was bad. Probably it was either that DTC threw some error, or that I couldn't get them to start in DOSBox. Eitherway, it's probably not an accurate thing to say that it's a bad dump with so little research, so I'll remove the tag until it can be said more definite that they are bad.